src/games/strategy/engine/framework/GameRunner2.java

```diff
@@ -731,29 +731,11 @@ private static boolean checkForLatestEngineVersionOut() {
         SwingUtilities.invokeLater(new Runnable() {
           @Override
           public void run() {
-            EventThreadJOptionPane.showMessageDialog(null, latestEngineOut.getOutOfDateComponent(false),
+            EventThreadJOptionPane.showMessageDialog(null, latestEngineOut.getOutOfDateComponent(),
```

Comment:
No disagreement, but it's code that I have not touched, leaving it out of scope. I could easily embark on many different cleanups in nearby code, there is no lack there. With that said, I thought we had a global lambda replace on deck, looks like re-running that should be done so we can have a dedicated PRs. PRs that wander around with many different changes are not easy to review, we've been optimizing for reviewablity for the most part.